### PR TITLE
fix: remove emoji from console reporter

### PR DESCRIPTION
**Pull request by shanejonas**
removed emoji, added some colour to keep our sanity

image

### CODE REVIEW

File: src/coverage.ts [hunk 1]
```diff
@@ -19,6 +18,7 @@ interface IOptions {
 }
 
 export default async (options: IOptions) => {
+  const results: any[] = [];
   const promises = options.schema.methods.map(async (method) => {
     if (options.skipMethods.includes(method.name)) {
       return Promise.resolve();
```
Comment (zcstarr) [inline src/coverage.ts]: async always returns promise, I think you should be able to just return here.

File: src/reporters/console.ts [hunk 2]
```diff
@@ -6,14 +8,16 @@ export default (callResults: any[]) => {
   callResults.forEach((call) => {
     if (call.error) {
       metrics.errors++;
-      console.log('⛔️', call.method, 'ERROR:', call.error)
+      console.log(colors.red.underline('Error:'), colors.cyan(call.method));
```
Comment (zcstarr) [inline src/reporters/console.ts]: nit. but 'Error: ', is missing a space after the colon, for consistency

Comment (shanejonas) [inline src/reporters/console.ts]:
```suggestion
      console.log(colors.red.underline('Error :'), colors.cyan(call.method));
```

**Pull request merged**